Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -2223,14 +2239,6 @@ private int GetNextTickerId()
             return Interlocked.Increment(ref _nextTickerId);
         }
 
-        /// <summary>
-        /// Increments the client ID for communication with the gateway
-        /// </summary>
-        private static int IncrementClientId()
-        {
-            return Interlocked.Increment(ref _nextClientId);
```

Comment:
It seems this comes from the first implementation, wonder what was the reason it incremented the client ID

Reply:
Hangover from 2012 when I think each new connection required a new unique ID. Not sure if it's relevant anymore.